tacs/solvers/BaseSolver.py

```diff
@@ -0,0 +1,157 @@
+"""
```

Comment:
Lower case python file names to stay consistent with rest of module (i.e. `basesolver.py`)